integration_tests/test_logical_assignment.py

```diff
@@ -0,0 +1,18 @@
+from lpython import i32, f64
+
+
+def test_logical_assignment():
+    _LPYTHON: str = "LPython"
+    s_var: str = "" or _LPYTHON
+    assert s_var == "LPython"
+
+    _MAX_VAL: i32 = 100
+    i_var: i32 = 0 and 100
+    assert i_var == 0
+
+    _PI: f64 = 3.14
+    f_var: f64 = 2 * _PI or _PI**2
+    assert f_var == 6.28
+
+
+test_logical_assignment()
```

Comment:
- Why is this test separate from the other? It seems to test `selective or` for strings, why is it called as `assignment`?

- Can you add a print before the asserts?

Reply:
I created a separate test for testing assignments through logical operations that we implemented. I am adding the print statements.

Reply:
Ok, Let's merge this test with the other integration test added in this PR.